### DISCUSSION TITLE
Tasmota-Shelly with ESP-xxxxxx wLAN-SSID

### DISCUSSION THREAD

**Discussion by GerdRuetten** [arendst/Tasmota#18045] [Q&A]
Dear all,

i flashed a Shelly 1PM with Tasmota 12.4.0.
The flash was successfully.

After that, i connect the Shelly with Power and it spinn up a wLAN with SSID "ESP-123456" and not with "Tasmota-123456".

It's not my first flashed Shelly but this Problem is new for me.

I do the flashing several times; always with erasing Option from Tasmotizer.

I can connect to this "ESP-wLAN" but nothing happens.

Could someone explain me the problem and the solution :-)

TIA

Best regards 

Gerd

**Comment by barbudor** [comment]
ESP-XXXXX is generally sign of bad flashing caused by either a bad adapter not delivering enough power on 3v3 or using bad flashing options/not supported software.
Tasmotizer should not be a problem so may be your adapter is.

**Reply by GerdRuetten** [reply]
Thank you!
I flashed 20 Shelly with this adapter.
Actually it was 2 bevor the problem Shelly and 3 after it

So I think, the adapter is not the problem.

I don't know

**Reply by barbudor** [reply]
Try flashing with tasmota web installer
https://tasmota.github.io/install/

**Reply by GerdRuetten** [reply]
Hi @barbudor 

i flash the shelly with the webinstaller.
The installer told me flashing are successfully.

But after i put the shelly to power, he spin up the "ESP-xxxxxx" wLAN.

I think the shelly is damage and i will send it back to the seller

**Comment by CarloDeali** [comment]
I was looking for an answer on this topic as well.
I had the same problems. The ESP-xxxxxx was acting on 192.168.4.2 but it dosent help.
I tried a different Tasmotiser (1.1a) and it worked ok.

**Comment by popy2k14** [comment]
Had an A1T cross flashed from tasmota -> tasmota minimal -> esphome and working for weeks.

Today i decided to brick it and flashed tasmota minimal over the esphome webinterface, dont ask why ;-)
To revive it, i opened it up, connected my USB-Serial to it and reflashed it with esphome.
Now it's working again.
